app/controllers/home_controller.rb

```diff
@@ -218,6 +218,29 @@ def tagged
     end
   end
 
+  def for_domain
+    @domain = Domain.first_by(domain: params[:domain])
+
+    @stories, @show_more = get_from_cache(domain: @domain.domain) do
+      paginate @domain.stories
+    end
+
+    @heading = params[:domain]
+    @title = "Stories submitted for #{domain.domain}."
+    @cur_url = for_domain_url(params[:domain])
+
+    @rss_link = {
+      title: "RSS 2.0 - For Domain #{tags_with_description_for_rss(@tags)}",
```

Comment:
Need to be updated to print the domain, also link on next line.

Please add a test for this endpoint - doesn't have to do anything clever, just add one Story with url `example.com` and title "Domain Example" and see that this page includes the title.